ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -186,10 +189,15 @@ export default defineComponent({
       const { data } = await fetchMhRegistration(getMhrInformation.value.mhrNumber)
       const currentOwnerGroups = data?.ownerGroups || [] // Safety check. Should always have ownerGroups
       // Create an ID to each individual owner for UI Tracking
+      // TODO: Remove after API updates to include the ID for Owners
       currentOwnerGroups.forEach(ownerGroup => {
+        ownerGroup.isCurrentOwnerGroup = true
         for (const [index, owner] of ownerGroup.owners.entries()) {
           owner.id = ownerGroup.groupId + (index + 1)
+          owner.isCurrentOwner = true
```

Comment:
I'm not sure this property is required? 
As we discussed i'm including `action` properties on anything Added or Removed, to keep it consistent with other scenarios that track added, removed and edited etc.

Reply:
Right, this is not needed. Removed.